app/data_model/answer_store.py

```diff
@@ -223,8 +228,10 @@ def remove(self, group_id=None, block_id=None, answer_id=None, group_instance=No
         :param group_id:
         :param answer_instance:
         :param group_instance:
+        :param location:
```

Comment:
Shall we remove the :param and :return: documentation if we aren't going to add detail otherwise it's just boilerplate noise